components/_global-navigation.scss

```diff
@@ -3,8 +3,29 @@ $nav-height: 49px;
 $nav-background-color: $color-white;
 
 $logo-width: 230px;
+$search-input-padding: 3px 4px;
 $notification-counter-size: 18px;
 
+@mixin visible-search-input {
```

Comment:
I don't like the idea of including mixins multiple times for styling the same elements.
Do you think it's possible to simplify that code somehow?